.git-blame-ignore-revs

```diff
@@ -30,3 +30,5 @@ e3900d2ba5c9f91a24a9ce34520794c8366d5c54
 30fb2c4abaaaa966999eab11674f25b18460e609
 # 2023-06-06 clang-format on Foreach / Multi-Tensor-Apply
 515c4279416f13fcc3c898e560f8ae8f15139a03
+# [optim][BE] split test file into logical parts: SWA, LR, optim
+a53cda1ddc15336dc1ff0ce1eff2a49cdc5f882e
```

Comment:
This could go in another PR but it didn't feel worth another PR....I can be easily convinced otherwise

Reply:
TIL this file exists and github uses it by default. Very cool!